src/main/webapp/src/app/map/map.component.ts

```diff
@@ -8,5 +8,23 @@ import { Component, OnInit } from '@angular/core';
 
 export class MapComponent implements OnInit {
   constructor() {  }
-  ngOnInit() {  }
+  zoom = 12
+  center!: google.maps.LatLngLiteral;
+  options: google.maps.MapOptions = {
+    mapTypeId: 'terrain',
+    zoomControl: false,
+    scrollwheel: false,
+    disableDoubleClickZoom: true,
+    maxZoom: 15,
+    minZoom: 8,
+  }
+
+  ngOnInit() {
```

Comment:
Nit - add a todo to replace navigator.geolocation with location passed from location component

Reply:
removed